ocs_ci/ocs/platform_nodes.py

```diff
@@ -1668,42 +1668,89 @@ def start_nodes(self, nodes):
             "Start nodes functionality is not implemented"
         )
 
-    def restart_nodes(self, nodes, wait=True):
-        raise NotImplementedError(
-            "Restart nodes functionality is not implemented"
-        )
+    def restart_nodes(self, nodes, timeout=540, wait=True):
```

Comment:
while restarting 2 nodes, one node tends to take longer to reach ready state (takes around 7 to 8 mins sometimes). Thus provided the timeout as 540 secs